wm/frame.go

```diff
@@ -451,10 +462,10 @@ func newFrame(c *client) *frame {
 	full := c.Fullscreened()
 	decorated := c.Decorated()
 	if full {
-		x = 0
-		y = 0
-		w = c.wm.x.Screen().WidthInPixels
-		h = c.wm.x.Screen().HeightInPixels
+		x = int16(c.wm.Screens()[c.wm.Active()].X)
```

Comment:
This is a lot of lookups to set 4 variables, perhaps the API changes above make it cleaner?

Reply:
indeed. made it one lookup.